File: 4107-find-missing-elements/find-missing-elements.py
class Solution:
    def findMissingElements(self, nums: List[int]) -> List[int]:
        r=[]
        miny,maxy=min(nums),max(nums)
        for i in range(miny, maxy):
            if not i in nums:
                r.append(i)
        return r

File: 4107-find-missing-elements/find-missing-elements.cs
public class Solution {
    public IList<int> FindMissingElements(int[] nums) {
        List<int> res = new List<int>();
        int miny = nums.Min();
        int maxy = nums.Max();
        for(int i = miny+1; i < maxy; i++)
        {
            if(!nums.Contains(i))
            {
                res.Add(i);
            }
        }
        return res;
    }
}